sydtest/LICENSE.md

```diff
@@ -16,6 +16,8 @@ Copyright (c) 2020-2022 Tom Sydney Kerckhove
 ### Conditions:
 
 * _The software under test is not used for commercial purposes._
+  **OR**
+  _The software under test is open-source software licensed according to an [OSI-approved Open Source license](https://opensource.org/licenses)._
```

Comment:
Hah, I was putting off using sydtest on my OSS because it was not super clear. Thanks for adding it!